dms/models/mixins_thumbnail.py

```diff
@@ -14,73 +14,82 @@ class Thumbnail(models.AbstractModel):
     _name = "dms.mixins.thumbnail"
     _description = "Thumbnail Mixin"
 
-    custom_thumbnail = fields.Binary(
-        string="Custom Thumbnail", attachment=False, prefetch=False,
-    )
+    custom_thumbnail = fields.Binary(string="Custom Thumbnail", attachment=False)
     custom_thumbnail_medium = fields.Binary(
-        "Medium Custom Thumbnail", compute="_compute_custom_thumbnail", prefetch=False,
+        "Medium Custom Thumbnail",
+        attachment=True,
+        compute="_compute_custom_thumbnail",
+        store=True,
     )
     custom_thumbnail_small = fields.Binary(
-        "Small Custom Thumbnail", compute="_compute_custom_thumbnail", prefetch=False,
-    )
-
-    thumbnail = fields.Binary(
-        compute="_compute_thumbnail", string="Thumbnail", prefetch=False,
-    )
-
-    thumbnail_medium = fields.Binary(
-        "Medium Thumbnail", compute="_compute_thumbnail", prefetch=False,
-    )
-
-    thumbnail_small = fields.Binary(
-        "SmallThumbnail", compute="_compute_thumbnail", prefetch=False,
+        "Small Custom Thumbnail",
+        attachment=True,
+        compute="_compute_custom_thumbnail",
+        store=True,
     )
+    thumbnail = fields.Binary(compute="_compute_thumbnail", string="Thumbnail",)
```

Comment:
Does these fields makes sense to be preserved now?

Reply:
Not all are used, but the ones that are, are used now only in form views, where records are computed 1 by 1 and thus there's no performance bottleneck.

Since they are not a deep problem in this new fix, I decided to keep them, just to keep the model more stable.